modules/twitch-notify/configs/config.json

```diff
@@ -0,0 +1,50 @@
+{
+    "filename": "config.json",
+    "content": [
+      {
+        "field_name": "live-message",
+        "default": "Hey, %streamer% is live on Twitch streaming %game%! Check it out at %url% !",
+        "type": "string",
+        "allowEmbed": false,
```

Comment:
Please allow users to set embeds by using `embedType()` from `/src/functions/helpers.js`.